src/topic/TopicMessageQuery.js

```diff
@@ -401,11 +408,25 @@ export default class TopicMessageQuery {
                         this._attempt < this._maxAttempts &&
                         this._retryHandler(error)
                     ) {
+                        const delay = Math.min(
+                            250 * 2 ** this._attempt,
+                            this._maxBackoff
+                        );
+                        console.log(
+                            `Error subscribing to topic ${
+                                this._topicId != null
+                                    ? this._topicId.toString()
+                                    : "UNKNOWN"
+                            } during attempt ${
+                                this._attempt
+                            }. Waiting ${delay} ms before next attempt`
```

Comment:
We should add the error message (but not stacktrace) at the end like we do for Java.